app/helpers/application_helper.rb

```diff
@@ -27,6 +27,10 @@ def user_header
   def is_active_topic(topic)
     topic == params[:topic]
   end
+
+  def current_user_owns_demand?(demand)
```

Comment:
I'm actually inclined to move this into the more generic `DemandForm` you've created by renaming it. If that's the canonical way to create or update a `Demand` then I'd say this belongs there. And test it!